### PR TITLE
Remove support for PNG generation

### LINKED ISSUE #205: move PNG generation out of mathjax-node
As per F2F, the PNG generation will be moved out of mathjax-node; there will be alternative modules to use.

Cf. the discussion in #174; see also #191.

### PR DESCRIPTION
**Pull request by pkra**
Fixes #205.

### CODE REVIEW

**Comment by pkra** [comment]
Should we make another release before v1.0 and other breaking changes (like #206)?

**Comment by pkra** [comment]
This is marked WIP because of the open questions in #205.

**Comment by pkra** [comment]
The only remaining question I had was whether https://github.com/mathjax/MathJax-node/blob/master/lib/mj-single.js#L587-L593 could/should be dropped or not. 

But that's a minor item I think (and could be done when #219 is decided) so I think this is ready to go for now.

**Comment by pkra** [comment]
Ahem, this is not quite ready. I need to remove all other PNG-related things (readme, examples in bin etc)

**Comment by pkra** [comment]
Ok, that's better. There are still some references in mj-page but that's for #206 to fix.

File: README.md [hunk 1]
```diff
@@ -18,10 +18,7 @@ Use
 
 to install mathjax-node and its dependencies.
 
-These API's can produce PNG images, but that requires the
-[Batik](http://xmlgraphics.apache.org/batik/download.html) library.  It 
-should be installed in the `batik` directory.  See the README file in that 
-directory for more details.
+These API's no longer produce PNG images but can easily be integrated into workflows using svg-to-png converters (e.g., png2svg, librsvg, batik).
```
Comment (dpvc) [inline README.md]: Do you really mean `png2svg` here?  That sounds like the wrong direction.

**Comment by dpvc** [comment]
Just to be clear, this also removes not only the PNG generation, but the ability to produce image tags that use remote SVG files as well.  I assume you want to move that to a post-processing phase, but wanted to make sure that it was part of the plan, since it has not been discussed anywhere else and isn't in the PR title.

File: README.md (hunk 1 above)
Comment (pkra) [inline README.md]: Thanks for catching that.

**Comment by pkra** [comment]
> Just to be clear, this also removes not only the PNG generation, but the ability to produce image tags that use remote SVG files as well. I assume you want to move that to a post-processing phase, but wanted to make sure that it was part of the plan, since it has not been discussed anywhere else and isn't in the PR title.

Right. I actually remember not being able to figure out if this code was still relevant but then deciding that it also fits more naturally in post-processing. I can pull out the commit if you prefer (I have to rebase anyway).

**Comment by dpvc** [comment]
>  I can pull out the commit if you prefer

No need.  Leave it in, I just wanted it to be in the record.

**Comment by pkra** [comment]
@dpvc I rebased (and addressed your comment). I'm a little insecure as to the rebase (lots of changes within mj-single) but I went through it twice and I didn't spot any bad changes from the resolution.

**Comment by dpvc** [comment]
There rebase looks good.  I'll merge.

**Pull request merged**